src/cryo_tank/tanktools.jl

```diff
@@ -157,19 +160,19 @@ function find_Q_time(t::Float64, fuse::Fuselage, fuse_tank::fuselage_tank, fuelt
             z = z0 + (zf - z0)/(tf-t0) * (t - t0)
 
             #Calculate heat rate at this point
-            Q = tankWthermal(fuse, fuse_tank, z, Mair, xftank, ifuel)
+            Q = tankWthermal(fuse, fuse_tank, z, TSL, Mair, xftank, ifuel)
         end
     end
     return Q
 end
 
 """
-    analyze_TASOPT_tank(ac_orig, t_hold_orig::Float64 = 0.0, t_hold_dest::Float64 = 0.0, N::Int64 = 50)
+    analyze_TASOPT_tank(ac, t_hold_orig::Float64 = 0.0, t_hold_dest::Float64 = 0.0, N::Int64 = 50)
 
 This function analyses the evolution in time of a cryogenic tank inside a TASOPT aircraft model.
```

Comment:
```suggestion
This function analyses the evolution in time of a cryogenic tank inside a TASOPT aircraft model, accounting for operational constraints (i.e., holding times and venting).
```

is that about right^?

Reply:
Right indeed